packages/frontend2/src/content/collections.ts

```diff
@@ -3,11 +3,13 @@ import { delegatedProjectsCollection } from './delegated-projects'
 import { eventsCollection } from './events'
 import { glossaryCollection } from './glossary'
 import { publicationsCollection } from './publications'
+import { zkCatalogDescriptionsCollection } from './zkCatalogDescriptions'
 
 export const collections = {
   publications: publicationsCollection,
   'delegated-projects': delegatedProjectsCollection,
   events: eventsCollection,
   authors: authorsCollection,
   glossary: glossaryCollection,
+  zkCatalogDescriptions: zkCatalogDescriptionsCollection,
```

Comment:
Can use snake case for collection name and directory? Tomek already changed the casing for `delegated-projects`.

Reply:
https://github.com/l2beat/l2beat/pull/4378/commits/d32ebf676dd68189134a4fd44838467669fbdca6